testsuite/smoke-test/src/rosetta.rs

```diff
@@ -124,27 +222,43 @@ async fn test_block() {
         .unwrap();
     assert_eq!(response.block.unwrap(), genesis_block_by_hash);
 
+    // Block 1 is always a reconfig with exactly 1 txn
+    let block_1 = get_block(&rosetta_client, chain_id, 1).await;
+    assert_eq!(1, block_1.transactions.len());
+    // Block metadata won't have operations
+    assert!(block_1.transactions.first().unwrap().operations.is_empty());
+    assert!(block_1.timestamp > genesis_block.timestamp);
+
+    // Block 2 is always a standard block with 2 or more txns
+    let block_2 = get_block(&rosetta_client, chain_id, 2).await;
+    assert!(block_2.transactions.len() >= 2);
+    // Block metadata won't have operations
+    assert!(block_2.transactions.first().unwrap().operations.is_empty());
+    // StateCheckpoint won't have operations
+    assert!(block_2.transactions.last().unwrap().operations.is_empty());
+    assert!(block_2.timestamp >= block_1.timestamp);
+
     // No input should give the latest version, not the genesis txn
     let request_latest = BlockRequest::latest(chain_id);
     let response = rosetta_client.block(&request_latest).await.unwrap();
     let latest_block = response.block.unwrap();
 
     // The latest block should always come after genesis
-    assert!(latest_block.block_identifier.index > genesis_block.block_identifier.index);
-    assert!(latest_block.timestamp > genesis_block.timestamp);
+    assert!(latest_block.block_identifier.index >= block_2.block_identifier.index);
+    assert!(latest_block.timestamp >= block_2.timestamp);
 
     // The parent should always be exactly one version before
     assert_eq!(
         latest_block.parent_block_identifier.index,
         latest_block.block_identifier.index - 1
     );
 
-    // There should be exactly one txn
-    assert_eq!(1, latest_block.transactions.len());
+    // There should be at least 1 txn
+    assert!(latest_block.transactions.len() > 1);
```

Comment:
Comment doesn't match code, use `>=` or update the comment.

Reply:
woops